pymongo/mongo_client.py

```diff
@@ -321,6 +321,14 @@ def __init__(
             Defaults to ``-1``, meaning no maximum. If maxStalenessSeconds
             is set, it must be a positive integer greater than or equal to
             90 seconds.
+          - `serverSelector`: (callable) Optional, user-provided function that
```

Comment:
`server_selector`, we only use camelCase for kwargs that are also URI options.

Reply:
Ah, I see... I noticed only some kwargs used camelCase but was unable to identify a theme (or didn't spend enough time trying to do so). Good to know how things are organized.